pyinjective/proto/injective/permissions/v1beta1/permissions_pb2.py

```
@@ -12,32 +12,38 @@
 _sym_db = _symbol_database.Default()
 
 
-from pyinjective.proto.gogoproto import gogo_pb2 as gogoproto_dot_gogo__pb2
 from pyinjective.proto.cosmos.base.v1beta1 import coin_pb2 as cosmos_dot_base_dot_v1beta1_dot_coin__pb2
+from pyinjective.proto.gogoproto import gogo_pb2 as gogoproto_dot_gogo__pb2
```

Ruff (0.8.2)

`pyinjective.proto.gogoproto.gogo_pb2` imported but unused

Remove unused import: `pyinjective.proto.gogoproto.gogo_pb2`

(F401)